flybywiresim/aircraft
# fix(a380x): Fix green dot speed for heavy weights and high altitudes; Fix FMS speed target for GD > CLB speed limit

Fixes #9183 

## Summary of Changes

- Fix extrapolation for green dot speed in PseudoFAC. Values which are not present in FCOM table are set to 260 (M0.853 at FL400)
- Fix FMS short term managed speed when green dot is above climb speed limit

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

**Tier 1 + Tier 2 test please** 

1. Configure aircraft with MTOW
2. Look for SID with speed restrictions < 250kts until ~30nm out (or as late as possible)
3. Set cruise flight level to as high as possible (REC MAX)
4. Set up aircraft and FMS as usual, perform flight

The following should hold:
- Green dot speed shouldn't exceed VMO/MMO at high cruise FL
- When climbing with speed restriction > green dot speed, the autoflight system should target at least green dot speed when in clean conf

The following should not happen: 
![image](https://github.com/user-attachments/assets/bb79dfe2-93ae-42bb-9fb2-ddf9d21392b8)
(magenta bug can be below green dot, but autothrust should target at least green dot)


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
